compiler/damlc/tests/src/DA/Test/DamlcMultiPackage.hs

```diff
@@ -75,8 +75,11 @@ main = do
   release <- locateRunfiles (mainWorkspace </> "release" </> "sdk-release-tarball-ce.tar.gz")
   withTempDir $ \damlHome -> do
     setEnv "DAML_HOME" damlHome True
-    -- Install sdk 0.0.0 into temp DAML_HOME
-    void $ readCreateProcess (proc damlAssistant ["install", release]) ""
+    -- Install sdk `env:DAML_SDK_RELEASE_VERSION` into temp DAML_HOME
+    -- corresponds to:
+    --   - `0.0.0` on PR builds
+    --   - `x.y.z-snapshot.yyyymmdd.nnnnn.m.vpppppppp` on MAIN/Release builds
+    void $ readCreateProcess (proc damlAssistant ["install", release, "--install-with-custom-version", sdkVersion]) ""
```

Comment:
The problem here was that `daml install <tar path>` tries to `resolveSdkVersionToRelease` the `.tar`'s SDK version (on today's failed `main` build, this was `2.9.0-snapshot.20231220.12517.0.vdbe2b021 `), which uses `getAvailableSdkSnapshotVersions` and `resolveReleaseVersionFromGithub`, but ofc neither knows about the release _we're currently building_